### PR TITLE
 Propose a component or pattern community page is missing a word

### PR DESCRIPTION
**Pull request by digitalcourtney87**
Added a correction to line 13 in line 14, adding the word 'the'.

### CODE REVIEW

File: src/community/propose-a-component-or-pattern/index.md.njk [hunk 1]
```diff
@@ -10,7 +10,8 @@ Anyone can propose a new component or pattern for the GOV.UK Design System.
 
 To be successful, proposals need to show that the component or pattern being suggested would be [useful and unique](/community/contribution-criteria).
 
-Follow 3 steps to propose a component or pattern for the Design System.
+- Follow 3 steps to propose a component or pattern for the Design System.
```
Comment (nickcolley) [inline src/community/propose-a-component-or-pattern/index.md.njk]: Hey @digitalcourtney87!

Thanks for this change it looks good but there's one thing that needs fixing before we can publish this.

The - and + characters are to represent what needs to be added can be removed.

You can edit the file again by using the following link: https://github.com/digitalcourtney87/govuk-design-system/edit/patch-1/src/community/propose-a-component-or-pattern/index.md.njk?pr=%2Falphagov%2Fgovuk-design-system%2Fpull%2F767

Let us know if you need any help with this change.

Comment (digitalcourtney87) [reply src/community/propose-a-component-or-pattern/index.md.njk]: Thanks @nickcolley I've updated!

**Comment by nickcolley** [comment]
Hey @digitalcourtney87,

Thanks for helping with this one but I think there was a mixup and you and someone else picked up this same issue.

We'll close this one out but if you want to contribute a different issue you can try this one:

https://github.com/alphagov/govuk-design-system/issues/754

Let us know if you want any help 👍 

Nick

**Comment by nickcolley** [comment]
Cheers for updating, I'm going to close this one out as it was resolved in another pull request :)

**Pull request closed without merging**